require.js

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
No we do not need it, must have added it while resolving the test issues. Removed the file